src/components/modals/NightOrderModal.vue

```diff
@@ -2,12 +2,12 @@
   <Modal
     class="night-reference"
     @close="toggleModal('nightOrder')"
-    v-if="modals.nightOrder && roles.size"
+    v-show="modals.nightOrder && roles.size"
```

Comment:
Why the change here? Using `v-if` is pretty important, otherwise the modal content will be loaded and present in the DOM at all times, causing extra, unnecessary demand on the CPU.

Reply:
If the modal doesn't remain loaded then the value of isMaximized is forgotten, which imo is too annoying for the user. Is there really any non-trivial load on the CPU from this?!

Reply:
Seems like the v-show on the VoteHistory is actually a mistake. :grimacing: It's supposed to be v-if, too.
I prefer not remembering maximised state for now or adding a few lines to remember it on a per-modal basis to having all modals be present in the DOM at all times. Keep in mind that the list of modals is only going to grow bigger, not smaller, so we have to make sure the site won't be eating more memory whenever we add a new feature.

Reply:
ok, i'll remove all the v-ifs (but i'm not able to detect any visible perf impact of the townsquare under any conditions!)